Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -194,6 +227,25 @@ final class GINIMultipageReviewControllerTests: XCTestCase {
         
     }
     
+    func testPageCellColors() {
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.multipagePageIndicatorColor = .black
+        giniConfiguration.multipagePageBackgroundColor = .red
+        let viewController = MultipageReviewViewController(documentRequests: imageDocumentRequests,
+                                                           giniConfiguration: giniConfiguration)
+        _ = viewController.view
+        
+        let cell = viewController
+            .collectionView(multipageReviewViewController.pagesCollection,
+                            cellForItemAt: IndexPath(row: 0, section: 0)) as? MultipageReviewPagesCollectionCell
+        
+        XCTAssertEqual(cell?.pageIndicatorLabel.textColor, giniConfiguration.multipagePageIndicatorColor,
+                       "page cell indicator color should match the one specified in the configuration")
+        
+        XCTAssertEqual(cell?.bottomContainer.backgroundColor, giniConfiguration.multipagePageBackgroundColor,
```

Comment:
These are actually two different tests, aren't they? One for the text color and one for the background.

Reply:
I always have the same doubt here, either create a different test for each one or use the same for both. My concern is the performance, since unit tests should always be fast.
What do you think?

Reply:
I think you should measure how long this test takes. Because, as always, I advise against premature optimisation. These tests are creating views so it's possible that they take a bit of time, but I suspect they are really fast to the point where optimisation doesn't make sense.

That being said, there are many other test classes where you initialize a full view controller for every test, so either way it's happening everywhere.

Even if it turns out that these tests are slow, you can fix that by reusing cells between tests. Although you will have to be careful to reset the cell at the end of a test.

In general, I think performance will not be a big issue here.